src/blog/article/PageContent.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { Document } from '@contentful/rich-text-types'
+import { Wrapper } from '../../components/wrapper/Wrapper'
+import { BlogHeader } from './BlogHeader'
+import { BlogContent } from './blogContent'
+import { BlogFooter } from './BlogFooter'
+import { ShareBarWrapper, DesktopShareBar } from './PageContent.styles'
+export interface Category {
+  title: string
+  hexColor: string
+}
+
+interface Thumbnail {
+  file: { url: string }
+  id: string
+}
+interface Author {
+  displayName?: string
+  thumbnail?: Thumbnail
+  jobTitle?: string
+}
+
+interface Article {
+  json: Document
+}
+export interface PageContentProps {
+  article: Article
+  title: string
+  datePublished: string
+  category: Category
+  author: Author
+}
+
+export const PageContent: React.FC<PageContentProps> = ({
+  article,
+  title,
+  datePublished,
+  category,
+  author,
+}) => {
+  const availableContent =
+    article.json.content[0].content[0] &&
+    article.json.content[0].content[0].nodeType === 'text'
+      ? article.json.content[0].content[0].value
+      : ''
+  const body = JSON.stringify(availableContent)
+  let location = ''
+  if (typeof window !== 'undefined') {
+    location = window.location.href
+  }
```

Comment:
@laij84 I think we should put this in context rather than accessing the `window.location` inside the component.

Reply:
I made all the other changes, so feel free to let me know what you prefer for this one and I'll implement it :)

Reply:
@thetami No worries, I chatted with @laij84 about it last night. The concern I had was around SEO, but if it's no different from previous implementation, let's not worry about it 😄